src/pages/About.mdx

```diff
@@ -109,16 +102,17 @@ $$
 
 </div>
 
-The parameters of this model fall into three categories: a time dependent infection rate $\beta(t)$ time scales of transition to a different subpopulation $t_l$, $t_i$, $t_h$, $t_c$, and age specific parameters $m_a$, $c_a$ and $f_a$ that determine relative rates of different outcomes.
-The latency time from infection to infectiousness is $t_l$, the time an individual is infectious after which he/she either recovers or falls severely ill is $t_i$, the time a sick person recovers or deteriorates into a critical state is $t_h$, and the time a person remains critical before dying or stabilizing is $t_c$.
-The fraction of infectious that are asymptomatic or mild is $m_a$, the fraction of severe cases that turn critical is $c_a$, and the fraction of critical cases that are fatal is $f_a$.
+The parameters of this model fall into three categories: (1) A time dependent infection rate $\beta(t)$ time scales of transition to a different subpopulation $t_l$, $t_i$, $t_h$, $t_c$, and age specific parameters $m_a$, $c_a$ and $f_a$ that determine relative rates of different outcomes; 
+(2) The latency time from infection to infectiousness is $t_l$, the time an individual is infectious after which he/she either recovers or falls severely ill is $t_i$, the time a sick person recovers or deteriorates into a critical state is $t_h$, and the time a person remains critical before dying or stabilizing is $t_c$; 
```

Comment:
This isn't really a 1,2,3... what is now under (1) is an overview, the two following points give detail. Pretty confusing wording on our part...

Reply:
I believe I understand this section more fully now. How about this edit...?

"The parameters of this model fall into three categories, which determine the relative rates of different outcomes: (1) A time-dependent infection rate  [B(t)]; (2) Time scales of transition to a different subpopulation [tl], [ti], [th], [tc]; and (3) Age-specific parameters [ma]. [ca], and [fa]. Here the latency time from infection to infectiousness is presented as [tl], the time an individual is infectious after which he/she either recovers or falls severely ill is [ti], the time a sick person recovers or deteriorates into a critical state is [th], and the time a person remains critical before dying or stabilizing is t[c]. While the fraction of infectious individuals that are asymptomatic or mild is [ma], the fraction of severe cases that turn critical is [ca], and the fraction of critical cases that are fatal is [fa]." 

Hopefully, this more clearly explains the parameter categories and the variables within them?

Reply:
yes, I like this!